translation/h/list/00_00/personality_voice_c00_03_00/translation.txt

```diff
@@ -36,150 +36,150 @@
 えっ、お尻？　ううん、ちょっとびっくりしただけ……=Huh, anal? It's fine, I was just a bit surprised.
-仕方がないわね……優しくお願い=I guess I have no choice... please be gentle.
+仕方がないわね……優しくお願い=I guess I have no choice... Please be gentle.
 もう、また？　君も好きねぇ……=Already, again... I like it too.
-アナルプレイに目覚めちゃうなんて、私たち変態よね♪=Exploring anal play, we're perverted. ♪
+アナルプレイに目覚めちゃうなんて、私たち変態よね♪=Exploring anal play, we're so perverted. ♪
 試してみたいの？　い、いいわよ……=You want to try that? A-alright...
 よくほぐしてからにしてね……=Just make sure to loosen it well before you do...
 お尻も可愛がってくれるの？　ふふ、ありがとう=You like my ass too? Fufu, thank you.
 そう言うと思っていたわ。もちろん準備はバッチリよ=I thought you'd say that. Of course, let's do it.
 ごめんなさいね、そっちはまだ少し心の準備が……=I'm sorry, I'm still not prepared for this...
 今日はダメ……中までキレイにしてないから……=I can't do it today... I'm not clean back there.
-そっちよりも、オマンコの方で楽しまない？=Rather than that, why not enjoy yourself with my pussy?
-ええ～、私のこのぐしゅぐしゅになったアソコを放っておくつもり？=Ehh~, are you just going to ignore my slimy pussy?
+そっちよりも、オマンコの方で楽しまない？=How about you play with my pussy instead?
+ええ～、私のこのぐしゅぐしゅになったアソコを放っておくつもり？=Ehh～ , are you just going to ignore my slimy pussy?
 まだダメ。君には早いわよ=Not yet. It's too soon for that.
 今日はお預けよ。指でいじらせてあげるから……=Let's not do this today. I'll let you touch instead...
 お尻の穴がガバガバになっちゃうからダーメ=You'll wreck my asshole!
-君のおチンポをお尻にあげちゃうのはもったいないわ……♪=It's shameful to put your cock in my ass...♪
-（処女）うっ……だ、大丈……ぶ……=Uu... I'm... O-okay...
-い……いいわ……よ……=Yeah... okay...
-きたぁ……これこれぇっ=Come on... that's it, that's it!
-//いただきまぁぁすっ=
-//（処女）あ、こ、こんな……っ=
-//ああっ、拡がる……=
-//んっ、待ってましたぁ……♪=
-//あっつぅぅぅい、おチンポ好きぃ=
-//（アナル処女）やだ、そんな、簡単に……っ=
-//んっ！　き、きつくない……？=
-//アナルも……好きぃ=
-//あぁぁ……お尻で君のおチンポ食べてるぅ=
-//（アナル処女）入って……入ってくるぅ……！=
-//あうっ……力……抜かなきゃ……=
-//ダメ……感じちゃう=
-//早く……最後まで入れて……！=
-//（処女）ふふ……とうとう繋がっちゃったわね……=
-//私たち、またひとつになってるのね……=
-//あぁ、おっきいの……私の中でパンパンになってる……=
-//うふふ、君のおちんちんが私の中でビクビクしてる=
-//大丈夫……そんなに痛くないから……うふふ、幸せ……=
-//君とひとつになれる幸せ、感じてるわ……=
-//あぁ、もうイッちゃいそう……君のおちんちん気持ちよすぎよ……=
-//私のアソコ、君の形になっちゃった……ふふ、嬉しい？=
-//この瞬間、もうたまんない……！=
-//ねぇ、私のオマンコが喜んでるの、わかる？=
-//（処女）全部入ってる……なんだか感動しちゃった……=
-//もう、そんなにがっつかないの……=
-//うふふ、我慢できなかったのね♪　わかるわ……=
-//いつ入れられるかわからないっていうのもイイわぁ……♪=
-//君のペースでいいわよ……好きにしてちょうだい……=
-//合体完了ね……♪　しばらく離さないわよ=
-//私の中で君のおチンポがどんどんおっきくなってく……=
-//（アナル処女）すごい……初めてなのに、入っちゃったわ……=
-//そんなに……ココがいいのね……=
-//君のせいでもうすっかり慣れちゃった=
-//ねぇ、そんなにお尻が好きなの？　どうイイのか言ってみてよ=
-//私のお尻……どう？　喜んでもらえると、嬉しい……わ……=
-//アナルで感じちゃうなんて……私って変態なのかも♪=
-//アソコとは違った快感……ふふっ、すっかりクセになっちゃったわ=
-//（アナル処女）もう……いきなりすぎ……っ=
-//そんなに欲しかったの……気付かなくてごめんなさい=
-//もう入れちゃったのね……せっかちなんだから=
-//この違和感がイイわ……ねぇ、早く抜き差ししてみて……=
-//無理……じゃないけど……頑張る……わ……=
-//んもう……準備、まだだったのにぃ……=
-//おチンポ、お尻に刺さってるわよ……もう、わざとね=
-//私のアナル、オマンコになっちゃったみたい……すっごく感じるの……=
-//あぁ……しっかりと君を感じるわ……熱い……=
-//私の中、あったかいでしょ……？=
-//あぁ、もう、トロけちゃいそう……=
+君のおチンポをお尻にあげちゃうのはもったいないわ……♪=It's shameful to put your cock in my ass... ♪
+（処女）うっ……だ、大丈……ぶ……=(Virgin) Uu... I'm... O-okay...
+い……いいわ……よ……=Yeah... Okay...
+きたぁ……これこれぇっ=Come on... That's it, that's it!
+いただきまぁぁすっ=Thank you for the meal!
+（処女）あ、こ、こんな……っ=(Virgin) So, this is...
```

Comment:
Again, since the field these are from weren't originally displayed I think the stuff in `()` is just a note. Can we put comments on the same line as a translation? If so maybe something like this would be better? 
```
（処女）あ、こ、こんな……っ=So, this is... // (Virgin) 
```